rrfs-test/CMakeLists.txt

```diff
@@ -68,47 +68,77 @@ if(MPAS_DYCORE)
 
       get_all_keys(rrfs_mpasjedi_tests caselist)
       foreach(case ${caselist})
-	 get_from_dictionary(rrfs_mpasjedi_tests case exe)
+	       get_from_dictionary(rrfs_mpasjedi_tests case exe)
          set(casedir "${CMAKE_CURRENT_BINARY_DIR}/rundir-${case}")
          set(src_casedir "${rrfs-test_data_local}/rrfs-data_mpasjedi_2024052700")
          if (NOT EXISTS "${casedir}")
            file(MAKE_DIRECTORY ${casedir})
          endif()
-	 file(CREATE_LINK ${src_casedir}/data ${casedir}/data SYMBOLIC)
-	 file(CREATE_LINK ${src_casedir}/graphinfo ${casedir}/graphinfo SYMBOLIC)
-	 file(CREATE_LINK ${src_casedir}/stream_list ${casedir}/stream_list SYMBOLIC)
-	 file(CREATE_LINK ${CMAKE_SOURCE_DIR}/rrfs-test/testoutput ${casedir}/testoutput SYMBOLIC)
-	 file(COPY ${src_casedir}/streams.atmosphere DESTINATION ${casedir})
-	 file(COPY ${src_casedir}/namelist.atmosphere DESTINATION ${casedir})
-	 file(COPY ${src_casedir}/geovars.yaml DESTINATION ${casedir})
-	 file(COPY ${src_casedir}/keptvars.yaml DESTINATION ${casedir})
-	 file(COPY ${src_casedir}/obsop_name_map.yaml DESTINATION ${casedir})
-	 file(GLOB bl_FILES "${src_casedir}/*.*BL")
-	 file(COPY ${bl_FILES} DESTINATION ${casedir})
-	 file(COPY ${src_yaml}/${case}.yaml DESTINATION ${casedir})
+	       file(CREATE_LINK ${src_casedir}/data ${casedir}/data SYMBOLIC)
```

Comment:
It looks like we get extra indentations from the latest commit. Are these changes expected?

Reply:
Yeah not sure what happened there. I used Github to do the merge conflicts and it added some weird spacing. Just fixed it, I think.